jobs/FunctionTest/FunctionTest.groovy

```diff
@@ -98,30 +139,40 @@ def functionTest(String test_name, String label_name, String TEST_GROUP, Boolean
                             // according to the member variable: TESTS
                             stash name: "$test_name", includes: "$artifact_dir/*.*"
     
+                            // Clean up test stack
+                            sh '''#!/bin/bash -x
+                            ./build-config/jobs/function_test/cleanup.sh
+                            '''
+
+                            // [Based on junit xml log] Write test results to github
                             sh '''#!/bin/bash -x
```

Comment:
There will be 2 "Shell Script" in log under pipeline view.
And it's  difficult to tell the difference of the 2 "Shell Script"
I  think it's better to combine the 2 ```sh``` steps into 1 step.